dynawo/sources/Common/Dictionaries/DYNError_en_GB.dic

```diff
@@ -47,6 +47,7 @@ SolverSIMConvFailMin        =             solverSIM fails to converge while mini
 SolverSIMConvFail           =             solverSIM fails to converge while the max number of try (%1%) is reached
 SolverEmptyYVector          =             impossible to create the KINSOL's structure : no continuous variable in simulated problem
 SolverUnstableZMode         =             discrete events are unstable during the solve of the equation
+SolverUnbalanced            =             the number of algebric variables is different from the number of algebric equations in the simulated problem
```

Comment:
Two small remarks:
- there is a typo all over the code in Dynawo but the correct word is "algebraic". We can begin correcting it by small step.
- the error could be raised on algebraic or differential variables/equations (we solve the problem partially on both subset of equations). I would suggest "the number of algebraic/differential variables ... algebraic/differential equations".

Reply:
Solved